tests/test_preload_script.py

```diff
@@ -19,27 +19,6 @@
                           send_JSON_command, subscribe)
 
 
-@pytest.mark.asyncio
-async def test_preloadScript_add_nonExistingContext_exceptionReturned(
```

Comment:
`context_id` should also be removed from preload script calls in other tests in this file